README.md

```diff
@@ -1,83 +1,166 @@
-# Keyworker UI App
+# OMiC UI / Keyworker UI App
 
-Application can be built with for dev mode
+This service is referred to by both names *OMiC UI* and *Keyworker UI*  (confusingly).
 
-```bash
+The service requires the following minimum tool versions:
+
+- yarn v1.17+ 
+- node v10+ 
+- gradle v5.6+
```

Comment:
this is not a requirement, we bundle the gradle wrapper with the project (currently at version 5.0)

Reply:
Clarified this.